documentation/modules/about-hook-crs-for-migration-plans-api.adoc

```diff
@@ -45,5 +49,51 @@ spec:
             name: playbook
           step: PreHook
 ----
+////
+
+The illustration that follows shows the general process of using a migration hook. Specific procedures are described in TBD1 and TBD2.
+
+[Figure]
+
+
+Process:
+
+
+. Input your Ansible hook and credentials.
+
+** Input an Ansible pre-migration hook image or an Ansible post-migration hook image to the {project-short} controller using the CLI. This step is always done using the CLI, whether the rest of the migration is done using the CLI or the UI.
+
+// Check text vs this content
+ can either specify the playbook in the Hook CRD or in the image itself. If the user specifies it in the image we use the playbook.yml and we set it to run it with the ansible-runner. Otherwise we keep it up to the users image what playbook is run but that needs to be specified in the image itself
+
+
+** If you need additional data to run the playbook inside the pod, create a Secret that contains credentials for the VM. The Secret is not mounted to the pod, but is called by the playbook.
+
+. The {project-short} controller creates the `configmap`, which contains:
+
+** `workload.yml`, which contains information about the VMs.
+** `playbook.yml`, the raw string playbook you want to execute.
+** `plan.yml`, which is the `Plan CR`.
++
+The `configmap` contains the name of the VM and instructs the playbook what to do.
+
+. The {project-short} controller does the following:
+
+** Creates a job that starts the user specified image.
+** Mounts the `configmap` to the container.
+
++
+The Ansible hook imports the Secret that the user previously
+. The job runs a prehook or a posthook as follows:
+
+** For a prehook, the job logs into the VMs on the source provider using SSH and runs the prehook.
+** For a posthook, the job logs into the VMs on {virt} using SSH and runs the posthook.
+
+// Next, 1 or 2 procedure modules based on the blog.
+
+
```

Comment:
When you are ready, please, go back and remove any additional and un-needed lines, inline comments, and block comments

Thanks